packages/cli/src/cmds/beacon/initBeaconState.ts

```diff
@@ -44,6 +70,16 @@ export async function initBeaconState(
     return await initFromFile(args.weakSubjectivityStateFile);
   } else if (dbHasSomeState) {
     return await initStateFromDb(config, db, logger);
+  } else if (args.fetchWeakSubjectivityStateFromIPFS) {
+    if (args.network === "mainnet") {
+      return await initAndVerifyWeakSujectivityState(args, mainnetWeakSubjectivityState, initFromFile, logger);
+    } else if (args.network === "prater") {
+      return await initAndVerifyWeakSujectivityState(args, praterWeakSubjectivityState, initFromFile, logger);
+    } else if (args.network === "pyrmont") {
+      return await initAndVerifyWeakSujectivityState(args, pyrmontWeakSubjectivityState, initFromFile, logger);
+    } else {
+      throw new Error("No matching network with weak subjectivity state.");
+    }
```

Comment:
Can be simplified by getting the `weakSubjectivityState` and then calling 
```ts
return await initAndVerifyWeakSujectivityState(args, weakSubjectivityState, initFromFile, logger);
```
once